benchmarks/bench_topk.py

```
@@ -79,10 +80,60 @@ def bench_flashinfer_modes(
+    ("small", TopKTieBreak.SMALL),
+    ("large", TopKTieBreak.LARGE),
+)
+
+
+def bench_tie_break_variants(
+    run_flashinfer_with_tie_break, baseline_ms: float
+) -> dict[str, float]:
+    metrics: dict[str, float] = {}
+    for suffix, tie_break in TIE_BREAK_VARIANTS:
+        try:
+            tie_ms = bench_median_ms(lambda: run_flashinfer_with_tie_break(tie_break))
```

Ruff (0.15.10)

[warning] Function definition does not bind loop variable `tie_break`

(B023)